dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -78,7 +78,7 @@ def get_supported_aggregation(self, series):
 
             if pd.api.types.is_string_dtype(series.dtype):
                 # If dask_cudf strings dtype, return built-in aggregation
-                if "cudf" in str(series._partition_type):
+                if is_cudf(series._partition_type):
```

Comment:
Would probably name this something like `is_cudf_type` to match up with Dask's similar utility functions; also wonder if it would make sense to add handling in the utility for Dask-cuDF and cuDF dataframes, so that here we would have a call like

```suggestion
                if is_cudf_type(series):
```

And the utility itself would check `_partition_type` if it sees that `series` is a Dask object?